.codeclimate.yml

```diff
@@ -15,4 +15,5 @@ exclude_patterns:
 - db/
 - vendor/
 -  "**/vendor/**/*"
-- app/assets/images/
\ No newline at end of file
+- app/assets/images/
+- spec/**/*
```

Comment:
👍 nice, even codeclimate recommends to [exclude specs/tests](https://docs.codeclimate.com/docs/excluding-files-and-folders#auto-generated-file-and-folder-exclusions) from the quality check